Dockerfile

```diff
@@ -87,12 +87,12 @@ RUN apt update -qq && apt install -qq -y --no-install-recommends \
     && gem install bundler \
     && rm -rf /var/lib/apt/lists/*;
 
-# install nodejs and yarn packages from nodesource
-RUN curl -sL https://deb.nodesource.com/setup_${NODE_VERSION} | bash - \
-    && apt-get update -qq \
-    && apt-get install -qq -y --no-install-recommends nodejs \
-    && npm i -g yarn \
-    && rm -rf /var/lib/apt/lists/*
+# install nodejs using n
+RUN curl -L https://raw.githubusercontent.com/tj/n/master/bin/n -o n \
+    && bash n lts \
```

Comment:
why not just install Node.js once ? This would save us build time.

Reply:
done